DirectEmployers/MyJobs
# [PD-3282] [PD-3393] Company Impersonate Logic, Analytics

## What's this PR do?
This PR adds a ajax-select control to the analytics platform to allow a user with the "impersonate analytics" activity to impersonate another company and view their analytics.

## Screenshots (if appropriate)

## Where should the reviewer start?
Any analytics view.
https://secure.my.jobs/analytics/view/main/

## How should this be manually tested?
Using shell or the django admin, add the app access of "Impersonation" to a company that you are admin to.

After that, select that company in the topbar and head to Analytics.

There should be a dropdown in the upper left corner. Enter the name of a company that has analytics data (or one that doesn't).

Notice how the data displayed changes based on what company is selected.

## Any background context you want to provide?

I dunno. It's 4:43. Maybe I'll add more to this later.
